gdpc/direct_interface.py

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/python3
+"""### Provide access to the HTTP interface of the Minecraft HTTP server.
+
+This file contains various functions that map directly onto the HTTP interface.
+It is recommended to use `interface.py` instead.
+"""
+__all__ = []
+__version__ = "v4.3_dev"
+
+import requests
+from requests.exceptions import ConnectionError
+
+
+def getBlock(x, y, z):
+    """**Return the name of a block from the world**."""
```

Comment:
I would change "name of a block" to "type of a block" or "name of a block's type". I find it potentially confusing as it is.

Reply:
Changed that to 'block ID' everywhere